### PR TITLE
Navigation Addded as component

### PR DESCRIPTION
**Pull request by adilatilganallocadia**
@alpatilgan I have added navigation as requested in  #5 .
Navigation will be used for user to navigate through our app.

### CODE REVIEW

File: src/app/app.component.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<!--The content below is only a placeholder and can be replaced.-->
+<app-nav></app-nav>
```
Comment (alpatilgan) [inline src/app/app.component.html:1]: Why did you add navigation?

Comment (adilatilganallocadia) [reply src/app/app.component.html:1]: Navigation is the place for our menu, so users can select which page to go. 
Like 
Home . PlayList . About . Contact

**Comment by alpatilgan** [comment]
You should resolve conflict

**Comment by adilatilganallocadia** [comment]
Conflict resolved

**Review by alpatilgan — APPROVED** [review]
